docs/src/pages/en/responsive-controls.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Responsive controls
+description: Learn how to make controls responsive with Media Chrome 
+layout: ../../layouts/MainLayout.astro
+---
+
+One thing you'll quickly notice is that Media Chrome is not responsive out of
+the box. This is by design, there are way too many permutations possible to
+provide one configuration for a responsive media player. However media themes
+will be introduced soon that can provide responsive behavior by default.
+
+There are different techniques available to make your Media Chrome controls 
+responsive but we'll keep it simple in this guide and make use of 
+[`ResizeObserver`](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver)
+and CSS classes.
+
+## Basic unresponsive layout
+
+Lets create a basic layout for our Media Chrome player.
+
+```html
+<style>
+  media-controller {
+    display: block;
+    aspect-ratio: 16 / 9;
+  }
+  video {
+    width: 100%;
+  }
+</style>
+<media-controller>
+  <video
+    playsinline
+    slot="media"
+    src="https://stream.mux.com/O6LdRc0112FEJXH00bGsN9Q31yu5EIVHTgjTKRkKtEq1k/high.mp4"
+    poster="https://image.mux.com/O6LdRc0112FEJXH00bGsN9Q31yu5EIVHTgjTKRkKtEq1k/thumbnail.jpg?time=56"
+  ></video>
+  <media-control-bar>
+    <media-play-button></media-play-button>
+    <media-mute-button></media-mute-button>
+    <media-volume-range></media-volume-range>
+    <media-time-display></media-time-display>
+    <media-time-range></media-time-range>
+    <media-duration-display></media-duration-display>
+    <media-playback-rate-button></media-playback-rate-button>
+    <media-fullscreen-button></media-fullscreen-button>
+  </media-control-bar>
+</media-controller>
+```
+
+<style>
+  media-controller {
+    display: block;
+    aspect-ratio: 16 / 9;
+  }
+  video {
+    width: 100%;
+  }
+</style>
+<media-controller id="mc1">
+  <video
+    playsinline
+    slot="media"
+    src="https://stream.mux.com/O6LdRc0112FEJXH00bGsN9Q31yu5EIVHTgjTKRkKtEq1k/high.mp4"
+    poster="https://image.mux.com/O6LdRc0112FEJXH00bGsN9Q31yu5EIVHTgjTKRkKtEq1k/thumbnail.jpg?time=56"
+  ></video>
+  <media-control-bar>
+    <media-play-button></media-play-button>
+    <media-mute-button></media-mute-button>
+    <media-volume-range></media-volume-range>
+    <media-time-display></media-time-display>
+    <media-time-range></media-time-range>
+    <media-duration-display></media-duration-display>
+    <media-playback-rate-button></media-playback-rate-button>
+    <media-fullscreen-button></media-fullscreen-button>
+  </media-control-bar>
+</media-controller>
+
+That looks pretty good but the controls are static no matter the size of the player. 
+Lets change that, for this exercise we'll make a big center play button when the player 
+width is smaller than `484px` and only show the timerange at the bottom of the player. 
+
+## Observe player resizing
+
+This will require the [`ResizeObserver`](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver)
```

Comment:
suggestion(non-blocking):
> Since we're still waiting with bated breath for [CSS Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) to get [wider adoption](https://caniuse.com/css-container-queries), this will require the ...

Reply:
I'll change the links but I didn't know **bated breath**, maybe best keep it simple